code/game/objects/items/weebstick.dm

```diff
@@ -0,0 +1,208 @@
+	throw_speed = 3
+	throw_range = 7
+	sharpness = IS_SHARP_ACCURATE
+	embedding = list( "pain_mult" = 0, "jostle_pain_mult" = 0, "embed_chance" = 300, "armour_block" = 100, "fall_chance" = 0, "ignore_throwspeed_threshold" = TRUE)
+	// Last ditch effort to kill someone. Obviously, terrible idea unless the situation is desperate, as it WILL embed and WILL need surgery to remove. No extra embed damage tho, so as to not make it op.
+	hitsound = 'sound/weapons/anime_slash.ogg'
+	armour_penetration = 40
+	block_level = 2
+	block_upgrade_walk = 0 // Base desword blocking, but no possible improvements.
+	block_power = 50 //lower block power tho.
+	block_sound = 'sound/weapons/egloves.ogg' //that's a terrible parry sound
+	block_flags = BLOCKING_ACTIVE | BLOCKING_PROJECTILE
+	max_integrity = 200
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 70, "stamina" = 0)
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "diced", "cut")
+	var/primed = FALSE // necessary for anime bullshit, determines dash attack readiness
+	// it's cooler to put it back into the sheathe then pull it out into a dash *teleports behind u, for peak animuh
+	var/mob/living/owner // necessary to proc predash through on_enter_storage, which doesn't give the user. equipped doesn't work here.
+	var/dash_sound = 'sound/weapons/unsheathed_blade.ogg'
+	var/beam_effect = "blood_beam"
+	var/phasein = /obj/effect/temp_visual/dir_setting/cult/phase
+	var/phaseout = /obj/effect/temp_visual/dir_setting/cult/phase
+	COOLDOWN_DECLARE(katanalinking_cd)
+
+/obj/item/weebstick/Initialize()
+	. = ..()
+	AddComponent(/datum/component/butchering, 25, 90, 5) //Not made for scalping victims, but will work nonetheless
+
+/obj/item/weebstick/suicide_act(mob/user)
+	if(prob(50))
+		user.visible_message("<span class='suicide'>[user] carves deep into [user.p_their()] torso! It looks like [user.p_theyre()] trying to commit seppuku...</span>")
+	else
+		user.visible_message("<span class='suicide'>[user] carves a grid into [user.p_their()] chest! It looks like [user.p_theyre()] trying to commit sudoku...</span>")
+	return (BRUTELOSS)
+
+/obj/item/weebstick/attack(mob/living/victim)
+	if(iscarbon(victim))
+		var/prevstat = victim.stat //save original state
+		. = ..() // run attack
+		if(victim.mind && victim.stat == SOFT_CRIT && victim.stat != prevstat) //when they're in crit but they weren't before the attack, prime it
+			primed = TRUE
+	else
+		var/prevstat = victim.stat
+		. = ..()
+		if(victim.mind && victim.stat == DEAD && victim.stat != prevstat) //sadly silicons/simplemobs have to die.
+			primed = TRUE
+
+/obj/item/weebstick/attack_self(mob/living/carbon/user)
+	. = ..()
+	if (user != owner)
+		COOLDOWN_START(src, katanalinking_cd, 5 SECONDS) // doesnt work ? no runtime/bug
+		owner = user
+		var/opposite_active_arm = BODY_ZONE_L_ARM
+		if(!owner.active_hand_index % 2)
```

Comment:
what

Reply:
Ah fuck, I forgot to fix that. Yeah, it's supposed to be BODY_ZONE_R_ARM, and i need to add a // explaining what the fuck this is. There's basically no direct function for getting the body zone of the opposite hand of the active hand, so i make do. Thanks kapu for suggesting this because i'd never have found how

If you're talking about the cooldown, I just tried to copy it from another thing, and didn't understand why it didn't work. Left it in because it didn't crash anything, apparently, and it'd allow someone to look at it in the future

Reply:
deleted the previous one - overdumped info that was unrelated to what you were asking.

`active_hand_index` isn't a bool, you can have more than 2 hands, the modulo returns 0 or 1 depending on if it's a right or left one.

Reply:
but you put a bang on it